erpnext/public/js/setup_wizard.js

```diff
@@ -22,6 +22,8 @@ function load_erpnext_slides() {
 					placeholder:__('e.g. "Build tools for builders"'), reqd:1},
 				{fieldname:'bank_account', label: __('Bank Account'), fieldtype:'Data',
 					placeholder: __('e.g. "XYZ National Bank"'), reqd:1 },
+				{fieldname:'domain', label: __('Domain'), fieldtype:'Select',
+					options:["Distribution", "Manufacturing", "Retail", "Services", "Education"], reqd:1},
```

Comment:
Add "Other" option too in both Company and this list. Also, did you confirm if this translates?